app/controllers/file_uploads_controller.rb

```diff
@@ -0,0 +1,38 @@
+class FileUploadsController < ApplicationController
+
+  CATEGORIES = [
+      'Spawning Success',
+      # 'Tagged Animal Assessment',
+      # 'Untagged Animal Assessment',
+      # 'Wild Collection'
+  ].freeze
+
+  def index
+  end
+
+  def new
+    @categories = [['Select One', '']] + CATEGORIES.map{|c| [c,c]}
+  end
+
+  def upload
+
+    @result = 'Error: Invalid category!' unless CATEGORIES.include?(@category)
+    @category = params[:category]
+    uploaded_io = params[:input_file]
```

Comment:
Do you want to set @category before checking it in CATEGORIES.include?

Reply:
I will include the change in the next PR that builds out the job data importing.